strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/PypiArtifactGenerator.java

```diff
@@ -191,10 +208,78 @@ private void createSourcePackageFiles(ZipOutputStream zos,
                                               topLevelPath).getBytes();
         createZipEntry(zos, sourcesPath, sourcesContent);
 
+        copyLicenseFiles(zos);
+
         String randomPath = eggDirectory + "/RANDOM.txt";
         TestFileUtils.generateFile(zos, byteSize, randomPath);
     }
 
+    private String getLicenseName() 
+            throws IOException
+    {
+        if (!ArrayUtils.isEmpty(licenses))
+        {
+            if (licenses.length > 1)
+            {
+                throw new IOException("PyPi doesn't support multiple licenses!");
+            }
+
+            return licenses[0].license().getName();
+        }
+
+        return LicenseType.UNLICENSE.getName();
+    }
```

Comment:
I believe this method could be complete dropped. You can simply use:
```
licenses[0].license().getName();
```
....wherever necessary.

Is Python strict about expecting a license? If no, then there is no need to be falling back to a license such as `Unlicense`.

Reply:
Method dropped.